client/src/Seed/Seed.js

```diff
@@ -81,28 +87,33 @@ class Seed extends Component {
   }
 
   iframeLoaded() {
-    this.channel = Channel.build({
-      window: this.iframe.contentWindow,
-      origin: this.props.origin,
-      scope: this.props.scope,
-      debugOutput: this.props.debugOutput,
-      onReady: () => {
-        this.ready = true;
-        this.props.onReady && this.props.onReady();
-      }
-    });
+    try {
+      this.channel = Channel.build({
+        window: this.iframe.contentWindow,
+        origin: this.props.origin,
+        scope: this.props.scope,
+        debugOutput: this.props.debugOutput,
+        onReady: () => {
+          this.ready = true;
```

Comment:
I do not see the reason to have it so.
If it would be part of the state, its change would trigger a re-render, which is not what I want.